src/pages/Confirmation/ConfirmationStyle.scss

```diff
@@ -0,0 +1,109 @@
+@import "../../pages/Index/index.scss";
+
+.LessonTypeDisplayIcon {
+  width: 250px;
+  height: 250px;
+  top: 24px;
+  background: #b1b0ff;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 15%;
+  border-color: #FFFFFF;
```

Comment:
i think you can put just put `white` instead of straight hex code

Reply:
facts